polaris.shopify.com/next.config.js

```diff
@@ -1,9 +1,31 @@
+/* eslint-disable require-await */
 /** @type {import('next').NextConfig} */
+
 const nextConfig = {
   reactStrictMode: true,
   eslint: {
     ignoreDuringBuilds: true,
   },
+  async headers() {
+    return [
+      {
+        source: '/api/:path*',
+        headers: [
+          {key: 'Access-Control-Allow-Credentials', value: 'true'},
+          {key: 'Access-Control-Allow-Origin', value: '*'},
+          {
+            key: 'Access-Control-Allow-Methods',
+            value: 'GET,OPTIONS,PATCH,DELETE,POST,PUT',
```

Comment:
```suggestion
            value: 'GET,OPTIONS',
```

Think we should follow the principle of least privilege here.

Reply:
Works for me 👍  updated